cumulus/polkadot-parachain/src/cli.rs

```diff
@@ -59,6 +59,15 @@ pub enum Subcommand {
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct ExtraArgs {
+	/// EXPERIMENTAL: Use slot-based collator which can handle elastic scaling.
+	///
+	/// Use with care, this flag is unstable and subject to change.
+	#[arg(long)]
+	pub experimental_use_slot_based: bool,
+}
```

Comment:
This struct should be an "extra struct". This means that it is not parsed directly from the CLI. This struct should be setup from the CLI, but not parsed directly.

Reply:
Ok, got it. Done. Moved the struct definition to `common`, renamed it to `NodeExtraArgs` and put the `experimental_use_slot_based` field back in the Cli struct.